src/components/Menu.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Disclosure, Transition } from '@headlessui/react'
+
+
+const Menu = (): JSX.Element =>  {
+    const navigation = ['Calendar', 'Events', 'Attendance'];
+    const {innerWidth: width} = window;
+
+    return (
+        <div className="flex items-center justify-between space-x-16 md:space-x-0 lg:space-x-0 min-w-100 h-24 bg-red-500 px-5">
+            <div>
+                <img src="https://images.squarespace-cdn.com/content/v1/5939fcd1db29d6ec60929205/1599605891670-HLWDP9UQBSK6XT6DLF3A/SGA+White+Text+Transparent.png%3Fformat=1500w" alt="Student Government Association Logo" className="w-32 "/>
+            </div>
+
+            <ul className="flex items-center justify-around space-x-8">
+                {
+                navigation.map((elem: string) => {
+                    return <li><a href="" className="text-white hover:text-black hidden sm:block sm:text-xl">{elem}</a></li>
```

Comment:
When using `.map()`, I recommend assigning a key prop to each of the elements you're returning. This key is useful for allowing React to uniquely identify each element in your map and update accordingly if needed

Reply:
The key prop will need to be a value that never changes and uniquely identifies each element - this can be something like the `elem` for each navigation item.